hooks/post_gen_project.py

```diff
@@ -3,27 +3,22 @@
 """
```

Comment:
Remove the file rather than commenting all of it.

Reply:
I deleted it.